stable/artifactory/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: artifactory
 home: https://www.jfrog.com/artifactory/
-version: 7.13.0
+version: 7.12.9
```

Comment:
@danielezer chart version should be 7.13.1.

Reply:
@jainishshah17 changed